packager/app/test/packager_test.py

```diff
@@ -593,6 +593,16 @@ def testPackageWithLiveProfileAndKeyRotationAndNonDashIfIop(self):
         self.mpd_output,
         'bear-640x360-av-live-cenc-rotation-non-iop-golden.mpd')
 
+  def testPackageWithEncryptionOfOnlyVideoStream(self):
```

Comment:
Can you move this function after line 242, i.e. after testPackageWithEncryption?